src/test/scala/org/kframework/TextToKoreTest.scala

```diff
@@ -231,13 +231,13 @@ class TextToKoreTest {
     parseFromFileExpectParseException("test-exception-8.kore")
   }
   @Test def test_exception_9(): Unit = {
-    parseFromFileExpectParseException("test-exception-9.kore")
+    parseFromFileExpectParseException("test-pattern-16.kore")
```

Comment:
The Scala parser is supposed to be in sync with the Kore one, so these files should not generate exceptions.

Reply:
I see that you mentioned that in the PR description, but I still think that they should stay in sync.

Reply:
What do you think is the best course of action?

Reply:
I see three options, I think 1 is the most reasonable:

1. Fix the Scala parser to accept these (i.e. to parse meta-sorts) and call parseFromFile here. It should be fairly simple, but I didn't look at that particular part of the code. That's the approach we used until now.

2. If 1 seems too hard (e.g. you don't know Scala and you think you can't figure it out enough to update the parser), you could try to convince Xiaohong to update it, since he wrote the initial code. Or someone else from the team.

3. Convince the other people that we should give up on the Scala parser for now (e.g. by deleting the code) and that we should rewrite it latter when we actually need it. You might some weak support from me, but you don't have that many chances since people are convinced that we'll need it, and updating it every now and then is less effort than reimplementing/updating it later.

Reply:
BTW, that someone else I mentioned before can be me, of course.

Reply:
It looks like the scala parser was out of sync already, https://github.com/kframework/kore/commit/7ba1247896aeb5409019944c14f1c4f9fe915c33 introduces new syntax and tests but they weren't added to scala.  I can fix this too but I'm afraid that this PR is getting stale...

Reply:
Ok, how about this: I'll approve this pull request in order to minimize merge pain, and you'll update the Scala parser to handle meta-sorts in a follow-up PR. Optionally you can either fix the 'next' and 'rewrites' issues, and change the way tests are being run, or you can make a Trello task for that.